src/components/Footer/index.js

```diff
@@ -1,34 +1,43 @@
 import React from "react";
-import { Row, Col, Divider } from "antd";
+// import { Row, Col, Divider } from "antd";
```

Comment:
If this line is unnecessary for the code, you can remove it.

Reply:
done!

Reply:
Nice!